README.rst

```diff
@@ -34,17 +34,17 @@ Usage Example
   import busio
   import digitalio
   import adafruit_max31856
-  
+
   # create a spi object
   spi = busio.SPI(board.SCK, board.MOSI, board.MISO)
-  
+
   # allocate a CS pin and set the direction
   cs = digitalio.DigitalInOut(board.D5)
   cs.direction = digitalio.Direction.OUTPUT
-  
+
   # create a thermocouple object with the above
   thermocouple = adafruit_max31856.MAX31856(spi,cs)
```

Comment:
The most pedantic comment I have left on a code review in possibly years:

```suggestion
  thermocouple = adafruit_max31856.MAX31856(spi, cs)
```

Reply:
Pedantic but accurate. 🌮